indexer/processors/bridge/l1_bridge_processor.go

```diff
@@ -155,12 +156,25 @@ func L1ProcessFinalizedBridgeEvents(log log.Logger, db *database.DB, metrics L1M
 		log.Info("detected proven withdrawals", "size", len(provenWithdrawals))
 	}
 
+	skippedOVM1Withdrawals := 0
 	for i := range provenWithdrawals {
 		proven := provenWithdrawals[i]
 		withdrawal, err := db.BridgeTransactions.L2TransactionWithdrawal(proven.WithdrawalHash)
 		if err != nil {
 			return err
 		} else if withdrawal == nil {
+			// Handle networks with OVM 1.0 state (OP Mainnet/Goerli) in order to gracefully skip these withdrawals
+			portalAddr := l1Contracts.OptimismPortalProxy
+			if portalAddr == config.Presets[10].ChainConfig.L1Contracts.OptimismPortalProxy || portalAddr == config.Presets[420].ChainConfig.L1Contracts.OptimismPortalProxy {
```

Comment:
I don't think goerli has ovm 1.0 state. I think we were using Kovan way back then

Reply:
ohh okay interesting, i'll just leave it in a sanity. The cross domain message nonce also starts at 100k for goerli so not harmful to include just incase

First message sent from OP-Goerli: https://goerli-optimism.etherscan.io/tx/0x254d9c28add020404142f840ed794cea51f86c0f0a737e3e7bdd7e1e4550962e#eventlog

Reply:
yea you're right, i checked the Goerli genesis timestamp in comparison with the regenesis state. Georli should just be OVM 2.0